protos/action/action.proto

```diff
@@ -185,6 +185,7 @@ message GotoLocationResponse {
     ActionResult action_result = 1;
 }
 
+// Yaw behaviour during orbit flight.
 enum OrbitYawBehavior {
     ORBIT_YAW_BEHAVIOR_HOLD_FRONT_TO_CIRCLE_CENTER = 0;
```

Comment:
It's missing the documentation next to the values, like [here](https://github.com/mavlink/MAVSDK-Proto/blob/main/protos/camera/camera.proto#L218-L222)

Reply:
It has been enhanced.